packages/contracts/test/plugin/plugin-setup-processor.ts

```diff
@@ -467,82 +547,119 @@ describe('Plugin Setup Processor', function () {
       });
 
       it('reverts if plugin is not applied yet', async () => {
-        const pluginSetup = pluginSetupV1Mock.address;
-
         const {plugin, helpers} = await prepareInstallation(
           psp,
           targetDao.address,
-          pluginSetup,
-          pluginSetupMockRepoAddress,
+          setupV1.address,
+          repoUUPS.address,
           EMPTY_DATA
         );
 
         await expect(
```

Comment:
I'd add this line as well before calling `prepareInstallation` to make sure that `prepareInstallation` mightn't be hacking it. So, this line before calling `prepareInstallation` and also after calling `prepareInstallation`